README.md

```diff
@@ -109,10 +109,11 @@ class My::Application < Rails::Application
   # The default is `false`
   config.browserify_rails.evaluate_node_modules = true
 
-  # Force browserify on every found JavaScript asset
+  # Force browserify on every found JavaScript asset if true.
+  # Can be a proc.
   #
   # The default is `false`
-  config.browserify_rails.force = true
+  config.browserify_rails.force = ->(file) { File.extname(file) == ".ts" }
```

Comment:
Only thing is the comment is kind of wonky -- should be here is an example of only forcing it on files that end with `.ts` but I'll merge and we can update README.